test/e2e/specs/editor/various/block-bindings.spec.js

```diff
@@ -1248,50 +1248,125 @@ test.describe( 'Block bindings', () => {
 					page.locator( '#paragraph-binding' )
 				).toBeHidden();
 			} );
+
+			test( 'should add empty paragraph block when pressing enter', async ( {
+				editor,
+				page,
+			} ) => {
+				await editor.insertBlock( {
+					name: 'core/paragraph',
+					attributes: {
+						content: 'paragraph default content',
+						metadata: {
+							bindings: {
+								content: {
+									source: 'core/post-meta',
+									args: { key: 'text_custom_field' },
+								},
+							},
+						},
+					},
+				} );
+				const paragraphBlock = editor.canvas.getByRole( 'document', {
+					name: 'Block: Paragraph',
+				} );
+				await paragraphBlock.click();
```

Comment:
`editor.insertBlock` already selects the block, no?

Reply:
You're totally right. I think I added that because in the buttons block we actually need to select the inner button, but it is not needed for the paragraph and heading. I changed that in [this commit](https://github.com/WordPress/gutenberg/pull/59361/commits/63ffff87ca048a218567772710dcc45affcfd2aa).